mathics/builtin/graphics.py

```diff
@@ -3142,7 +3142,7 @@ def get_range(min, max):
 
         return elements, calc_dimensions
 
-    def boxes_to_tex(self, leaves, **options):
+    def boxes_to_tex(self, leaves, forxml=False, **options):
```

Comment:
Adding a parameter like this feels really hacky. 

The front end should be involved here.  The backend merely needs to provide the underlying building blocks.

Reply:
OK, but for it, I would need to merge #1140 first...

Reply:
Let's wait until #1140 clears then.

Reply:
The idea of these modifications is to have a working graphics engine without changing the current core. Most of the code is there, and if the control variable is not set (or set to false), we get the previous (buggy) behavior. 
On the other hand, this does not clash against #1140, this is why I put this in a separated PR.

Reply:
> Adding a parameter like this feels really hacky.
> 
> The front end should be involved here. The backend merely needs to provide the underlying building blocks.
Regarding this, I put `forxml` as an optional parameter.